package.json

```diff
@@ -154,7 +154,9 @@
       ]
     },
     "nsis": {
-      "include": "build/installer.nsh"
+      "include": "build/installer.nsh",
+      "oneClick": false,
```

Comment:
I'm not sure what side-effects this will have (if any)? Can package still be installed without admin rights?

Reply:
It should just be as normal apart from the install wizard showing. According to the [config here](https://www.electron.build/configuration/nsis) and how I read it, then you would be asked for elevation if it was required (because `allowElevation` is true by default).

I'm not sure about any other possible path related issues in the rest of the application though. Unfortunately I couldn't get a local build working to try out the install on my own machine, and was hoping that the CI would spit out an installer I could try which it either doesn't or I couldn't find it.

Reply:
Ok, let's take this into 4.1 pre-releases so that it can be tested more easily.